utils/nctl/sh/assets/setup_shared.sh

```diff
@@ -467,15 +474,17 @@ function setup_asset_global_state_toml_for_node() {
                 migrate-into-system-contract-registry -s $STATE_ROOT_HASH -d "$STORAGE_PATH"/"$(get_chain_name)")
 
         GLOBAL_STATE_VALIDATOR_OUTPUT=$("$NCTL_CASPER_HOME"/target/"$NCTL_COMPILE_TARGET"/global-state-update-gen \
-                change-validators $VALIDATOR_SETUP_STRING -s $STATE_ROOT_HASH -d "$STORAGE_PATH"/"$(get_chain_name)") 
+                generic -s $STATE_ROOT_HASH -d "$STORAGE_PATH"/"$(get_chain_name)" $VALIDATOR_SETUP_CONFIG) 
```

Comment:
The two calls in this file become completely unnecessary now that the `validators` field is optional in the `global_state.toml` file - they were only here so that this field becomes populated. So you should be able to remove them completely, and remove the `validator_setup_config_file` function, too.

Reply:
Yes, you're right, with the new changes we don't need this call. I'll merge `dev` into this and take them out.